src/test/java/de/mediathekview/mlib/filmlisten/NewFilmlistReadTest.java

```diff
@@ -44,16 +42,16 @@ public NewFilmlistReadTest(final String aJsonName, final FilmlistFormats aFormat
 
   @BeforeClass
   public static void initTestData() throws URISyntaxException, IOException {
-    testData = FilmlistTestData.getInstance().createTestdataNewFormat();
-    testFileFolderPath =
-        Paths.get(NewFilmlistReadTest.class.getClassLoader().getResource(BASE_FOLDER).toURI());
+    testData = FilmlistTestData.getInstance().createTestdataNewFormat();  
   }
-
+  
   @Test
   public void testRead() throws IOException {
-    final Path testFilePath = testFileFolderPath.resolve(jsonName);
+    final Path testFilePath = new File(jsonName).toPath();
     final Optional<Filmlist> result = filmlistManager.importList(format, testFilePath);
 
+    System.out.println(result.isPresent());
+
     Assert.assertThat(result.isPresent(), CoreMatchers.is(true));
     Assert.assertThat(result.get().getFilms().size(), CoreMatchers.is(testData.getFilms().size()));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "result.isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3655)